<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLineEdit" name="custo_pvc_b"/>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Tipo 2</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_12">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Tipo 3</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Preço de Venda das Correias:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Tipo 1</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLineEdit" name="preco_est_1"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Número de Funcionários</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_13">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Custos de Produção:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Aluguel</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="label_16">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Água</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_17">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Custos Fixos:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="label_19">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Imposto Sobre venda</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Frete por Materiais</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lim_pvc_e"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lim_pvc_v"/>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="lim_pvc_b"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Limite de Gasto com Materiais:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="custo_pvc_v"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="custo_pvc_e"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>PVC Verde</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>PVC Verde</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>PVC Ergonométrico</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Custo dos Materiais:</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLineEdit" name="quant_est_1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>PVC Branco</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_9">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>-1</number>
       </property>
       <property name="text">
        <string>PVC Ergonométrico</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLineEdit" name="quant_est_2"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>PVC Branco</string>
       </property>
      </widget>
     </item>
     <item row="19" column="2">
      <widget class="QLineEdit" name="quant_est_3"/>
     </item>
     <item row="16" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="24" column="2">
      <widget class="QPushButton" name="btn_sair">
       <property name="text">
        <string>Sair</string>
       </property>
      </widget>
     </item>
     <item row="24" column="1">
      <widget class="QPushButton" name="btn_calc">
       <property name="text">
        <string>Calcular</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Quantidade de Esteiras Tipo 2</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_20">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Resultados:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLineEdit" name="num_func"/>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_22">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Quantidade de Esteiras Tipo 1</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="preco_est_2"/>
     </item>
     <item row="9" column="2">
      <widget class="QLineEdit" name="preco_est_3"/>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="imposto"/>
     </item>
     <item row="12" column="2">
      <widget class="QLineEdit" name="agua"/>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="aluguel"/>
     </item>
     <item row="15" column="0">
      <widget class="QLineEdit" name="frete"/>
     </item>
     <item row="18" column="2">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Quantidade de Esteiras Tipo 3</string>
       </property>
      </widget>
     </item>
     <item row="19" column="3">
      <widget class="QLineEdit" name="lucro"/>
     </item>
     <item row="18" column="3">
      <widget class="QLabel" name="label_24">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>Lucro</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
